fix: LEAP-1762: Add selectItems support

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/398 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12992947323)

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12992947323

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12997464248)
> Successfully merged:  15 files changed, 135 insertions(+), 30 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-leap-1762/bulk-select-all'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12997464248

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12999780960

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12999814046
